shared/network/2.x.md

```diff
@@ -329,11 +329,8 @@ In order for a {{ $names.company.lower }} device to get outside of the local net
 
 Each of these should work with outward only (and inward once outward connection established) firewall settings.
 
-Additionally, if the network your device is connecting to works with whitelisting, you should whitelist the following domains on Port `80` and `443`:
+Additionally, if the network your device is connecting to works with whitelisting, you should whitelist the following domains on `443`:
```

Comment:
Misread this as public URL port.

I think this needs to say 'you should whitelist the following domains for the relevant ports above:`.

As long as the fields include `pool.ntp.org`, `resin.io`, `docker.com`, etc. then they're good (but with the right ports for the right domains).